sphericart-jax/python/sphericart/jax/__init__.py

```diff
@@ -1,18 +1,77 @@
 import jax
+from packaging import version
```

Comment:
do we already have a runtime dependency on packaging?

Reply:
No, we should add it here: https://github.com/lab-cosmo/sphericart/blob/f7b5b710684a04b5895942927aaf79b85d7f53ca/sphericart-jax/pyproject.toml#L5C1-L5C13